solution/src/models.py

```diff
@@ -5,7 +5,7 @@
 from .conf import models_dict
 
 models = {
-    model_name: pipeline(task=value["task"], model=value["model_path"], tokenizer=value["model_path"])
+    model_name: pipeline(task=value["task"], model=value["model_path"], tokenizer=value["model_path"], device=0)
```

Comment:
Hey @uncle-alfer, this version launched with some errors:
<details>

```
RuntimeError: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx
```

</details>

Can you please try specifying device as `"cuda:0"` (not just 0) and use `pytorch/pytorch:*-cuda*-cudnn*-runtime` as base image, as we found them to be hassle-free when working with AWS GPU instances. Thanks!